PB-IDE-Tools/displayCcode.pb

```diff
@@ -0,0 +1,310 @@
+﻿;-Description   : Displays the commented C code from the PB compiler
+; OS            : Windows, Linux, Mac
+; English-Forum :
+; French-Forum  :
+; German-Forum  : https://www.purebasic.fr/german/viewtopic.php?f=10&t=30935
+; 
+; Author        : Sicro
+; Author second : mk-soft (Modified for C-Code)
+; Update        : 28.11.2021
+
```

Comment:
I don't like the author, author second and update lines and the minus character before word _Description_.